app/views/github_oauth.py

```diff
@@ -7,65 +7,64 @@
 from rest_framework.response import Response
 from rest_framework_simplejwt.tokens import RefreshToken
 
-from ..oauth import (convert_to_auth_token, generate_github_access_token,
-                     get_user_from_token, retrieve_github_user_info)
+from ..oauth import (
+    convert_to_auth_token,
+    generate_github_access_token,
+    get_user_from_token,
+    retrieve_github_user_info,
+)
 
 from ..serializers import UserSerializer
 
 load_dotenv()
 
 # GITHUB ID AND SECRET
-SOCIAL_AUTH_GITHUB_KEY = os.getenv('SOCIAL_AUTH_GITHUB_KEY')
-SOCIAL_AUTH_GITHUB_SECRET = os.getenv('SOCIAL_AUTH_GITHUB_SECRET')
+SOCIAL_AUTH_GITHUB_KEY = os.getenv("SOCIAL_AUTH_GITHUB_KEY")
+SOCIAL_AUTH_GITHUB_SECRET = os.getenv("SOCIAL_AUTH_GITHUB_SECRET")
 
 # OAUTH TOOLKIT ID AND SECRET
-CLIENT_ID = os.getenv('CLIENT_ID')
-CLIENT_SECRET = os.getenv('CLIENT_SECRET')
+CLIENT_ID = os.getenv("CLIENT_ID")
+CLIENT_SECRET = os.getenv("CLIENT_SECRET")
```

Comment:
Declare these values where they are needed, check that they exist before using them.